Nasa7/CacheProvider.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  CacheProvider.swift
+//  AbsolutelyNewNasa
+//
+//  Created by Nikita Shvad on 21.11.2021.
+//
+
+import Foundation
+
+protocol CacheProviderProtocol {
+    func retrieve (key: String) -> Data?
```

Comment:
```suggestion
    func retrieve(key: String) -> Data?
```
Лишних пробелов наставил